lib/executor/command.cpp

```diff
@@ -26,14 +31,35 @@ static unique_ptr<char *const []> convertArguments(const vector<string> &vec) {
 	return unique_ptr<char *const[]>((char *const *)result.release());
 }
 
+/*!
+ * @brief Resolves program path.
+ * @param command
+ */
+static optional<string> resolvePath(const string &program) {
+	if (program.find('/') != string::npos) {
+		return program;
+	}
+
+	const auto &path = Environment::get("PATH");
+	for (const auto location : LazySplit(path, ":")) {
+		const auto programPath = std::filesystem::path(location) / program;
+		if (std::filesystem::exists(programPath)) {
```

Comment:
Should check if the file exists and is executable instead of just exists? Can't find anything in the standard about it

Reply:
Ur right

Reply:
@K1ngmar this seems to still be unresolved right?

Reply:
whoops